tctl-snap/Makefile

```
@@ -0,0 +1,43 @@
+REPO_ROOT = $(CURDIR)
+
+SNAPCRAFT = $(shell which snapcraft)
+ifeq "$(SNAPCRAFT)" ""
+SNAPCRAFT = /usr/bin/snapcraft
```

The only way that `which` does not find `snapcraft` but it exists in `/usr/bin` is if the user changed their `PATH` variable to exclude `/usr/bin`. Am I mistaken? This does not seem logical to me. It might be better to raise an error if the path could not be found.

---

You're right, updated. Thanks!